api/src/wfl/executor.clj

```diff
@@ -404,6 +423,7 @@
 (defn ^:private terra-executor-to-edn [executor]
   (-> executor
       (util/select-non-nil-keys (keys terra-executor-serialized-fields))
+      (dissoc :description)
```

Comment:
Can you explain why we pull this out here?

Reply:
The user doesn't need to see it